mrc-ide/hint
# mrc-2608 Persist current language in browser storage

This PR reuses VUE subscribe plugin in root  to listen for changes in changeLanguage mutation, if language is changed, selected language will be persisted in browser storage.

How to test:
* change app language and refresh page, the chosen language should remain as the app language and should not revert to English, which is the default language.

## Type of version change
_Delete as appropriate_

Major - Minor - Patches - None

## Checklist
